frontend/package.json

```diff
@@ -4,19 +4,20 @@
 	"description": "",
 	"main": "webpack.config.js",
 	"scripts": {
-		"dev": "cross-env NODE_ENV=development webpack serve --progress",
-		"build": "webpack --config=webpack.config.prod.js --progress",
+		"i18n:generate-hash": "node ./i18-generate-hash.js",
+		"dev": "npm run i18n:generate-hash && cross-env NODE_ENV=development webpack serve --progress",
+		"build": "npm run i18n:generate-hash && webpack --config=webpack.config.prod.js --progress",
```

Comment:
can we use `prebuild` script here ?

Reply:
I tried adding prebuild but it doesn't run when you do yarn dev, yarn lint or playwright command.  The hashes will be generated or update when each of these command run, so I think we have to have hash gen as a separate command.

Reply:
Got it